WEB-INF/classes/uon/seng2050/assignment/model/Article.java

```diff
@@ -0,0 +1,90 @@
+package uon.seng2050.assignment.model;
+
+import io.seanbailey.adapter.Model;
+import java.util.Date;
+import java.util.UUID;
+
+public class Article extends Model {
+
+  private UUID id;
+  private String title;
+  private String body;
+  private String answer;
+  private int helpfulness;
```

Comment:
I totally forgot to mention this, but you'll need to use the class version of primitives on models, so that they can be null. So in this case, you should use:
```java
private Integer helpfulness;
```

Reply:
I might add an annotation later to change this functionality.